.dockerignore

```diff
@@ -1,31 +1,3 @@
-/.circleci
-/.git
-/.github
-/.gitignore
-/.travis.yml
-/CHANGELOG.md
```

Comment:
Doesn't this affect our release image?

Reply:
Not the content since we pick all files explicitly to add to an image. See corresponding production dockerfiles. It can affect build speed a little bit since more files are added to the Docker context but it should be milliseconds. Anyway, I removed the largest directories just in case.